admin/class-enzuzo-cookie-consent-admin.php

```diff
@@ -91,7 +91,7 @@ public function settings_menu_init() {
 		// The setup section
 		add_settings_section(
 			'enzuzo_cookie_consent_setup_settings_section',
-			__( 'Setup', 'enzuzo-cookie-consent' ),
+			__( 'Setup', 'cookie-consent-integration' ),
```

Comment:
I think we can keep this enzuzo-cookie-consent?  Wherever visible on WP itself (links, menu items, etc.) - I'd like the word "enzuzo" shown.